lib/usdUfe/ufe/UsdUIInfoHandler.h

```diff
@@ -51,18 +51,19 @@ class MAYAUSD_CORE_PUBLIC UsdUIInfoHandler : public Ufe::UIInfoHandler
     std::string              treeViewTooltip(const Ufe::SceneItem::Ptr& item) const override;
     std::string              getLongRunTimeLabel() const override;
 
-private:
-    void updateInvisibleColor();
+    // Helpers
 
-    // Note: the on-color-changed callback function is declared taking a void pointer
-    //       to be compatible with MMessage callback API.
-    static void onColorChanged(void*);
+    // Called from treeViewIcon() method to find the correct icon based on node type.
+    // Get the map (node type -> icon filename) of the supported icon types.
+    // Can be overridden by derived class to add to the map.
+    typedef std::map<std::string, std::string> SupportedTypesMap;
+    virtual SupportedTypesMap                  getSupportedIconTypes() const;
 
+protected:
+    // Derived classes can set this color to override the default invisible color.
```

Comment:
Made fInvisibleColor protected so it can be modified by derived class (as it is in MayaUsdUIInfoHandler).